finetuner-workflow/finetune-workflow.yaml

```diff
@@ -374,14 +449,16 @@ spec:
       resources:
         requests:
           memory: 128Gi
-          cpu: "8"
+          cpu: 8
+          nvidia.com/gpu: 1
         limits:
-          memory: 192Gi
-          cpu: "8"
+          memory: 128Gi
```

Comment:
I have a feeling that setting this to 32/64 CPUs, with 256 GB memory might solve the 8 GPU issue.

Reply:
No, it didn't. We did 704gb and 32 cores. It's not tied to RAM or CPU.